sample/src/main/java/ch/sonect/sdk/shop/integrationapp/CacheManager.kt

```diff
@@ -0,0 +1,44 @@
+package ch.sonect.sdk.shop.integrationapp
+
+import android.content.Context
+
+private const val MERCHANT_ID = "merchantid"
+private const val CLIENT_ID = "clientid"
+private const val CLIENT_SECRET = "clientsecret"
+private const val DEVICE_ID = "deviceid"
+private const val HMAC_KEY = "hmac"
+private const val ENV_KEY = "envKey"
+
+class CacheManager(context: Context) {
+
+    private val sharedPreferences = context.getSharedPreferences("shopsample", Context.MODE_PRIVATE)
+
+    var merchantId: String?
```

Comment:
Is it nullable? As I see it also has default value.

Reply:
done